test/src/nyc-bin.js

```diff
@@ -388,4 +389,25 @@ describe('the nyc cli', function () {
       })
     })
   })
+
+  describe('hooks', function () {
+    it('provides coverage for requireJS and AMD modules', function (done) {
+      var args = [bin, '--hook-run-in-context', process.execPath, './index.js']
```

Comment:
lovely! I'm definitely finding myself writing more and more tests that directly exercise the bin which I don't think is a bad thing -- for one catches regressions I've introduced in the actual command line parsing.